### DISCUSSION TITLE
Fan controll of the controllboard fan

### DISCUSSION THREAD

**Discussion by Remleyfpv** [bassamanator/Sovol-SV06-firmware#71] [General]
I dont know if it is possible but would be cool if the controll board fan is controlled by the MCU temp of the STM. But I cant find what PIN this fan would be. The pin header is right next to a controllable fan(part cooling fan) so that leaves me to belive that it is a controllable fan output not just a 24V port.

**Comment by bassamanator** [comment]
>The pin header is right next to a controllable fan(part cooling fan) so that leaves me to belive that it is a controllable fan output not just a 24V port.

Unfortunately, that is not the case. Not possible via software/config from what I've seen, and I've looked into this quite a bit.

Do contribute by creating a PR if you find a solution.

**Comment by Remleyfpv** [comment]
oke and just to make sure: is yours turned of always to? or is mine just broken?

**Comment by bassamanator** [comment]
Everyone's hotend cooling fan is always turned on for the SV06/Plus. It's not a controllable fan.

**Comment by bassamanator** [comment]
Sorry, the motherboard fan turns on when the TMC drivers come into play, I assume. I say 'I assume' because I haven't found the pin config for it, nor does it listed in the official Marlin source code. What I do know that the motherboard fan turns on *only* when I start printing.

**Comment by bassamanator** [comment]
[Here's](https://forum-drucktipps3d-de.translate.goog/forum/board/42-sovol/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp) a forum that contains a lot of information, however, I have not found any of controlling 2 or the 3 fans, only the part cooling fan is controllable. It's in German but that link is running through google translate.

**Comment by bassamanator** [comment]
Converted issue into discussion. Hopefully others can contribute.

**Comment by Remleyfpv** [comment]
I asked Sovol what the PIN number is and they gave me the following response: 
It's: Terminal 2.5 4-T 

Does this help? cause I have no idea what they are talking about haha.

**Comment by bassamanator** [comment]
@Remleyfpv can you iterate what your fan issue is? The motherboard enclosure fan won't turn off?

**Reply by Remleyfpv** [reply]
The Motherboard enclosure fan doesnt turn on at all.  But the reason I am in this discussion/thread is bc i want to controll that fan with klippper or at least that fan pin.

**Comment by joescalon** [comment]
For anyone seeing this in the future, the MCU fan seems to be controlled by the part cooling fan, at least on my SV06 plus. Ran input shaper and fan stayed off, ran 10% part fan and MCU fan turned on.

**Reply by indistinctTalk** [reply]
Yeah, this is my experience too. It's pretty bad given I'm often printing with a low part cooling fan speed.
